3rd-party/vcpkg-ports/grpc/vcpkg-cmake-wrapper.cmake

```diff
@@ -0,0 +1,2 @@
+list(REMOVE_AT ARGS 0)
+_find_package(gRPC ${ARGS}) # Shouldn't this be fixed downstream instead of using a Wrapper?
```

Comment:
```suggestion
_find_package(gRPC ${ARGS}) # Shouldn't this be fixed downstream instead of using a Wrapper?

```

Reply:
Sorry, I did not see the difference between before and after.

Reply:
It adds a newline at the end, which is standard practice for text files (helps terminals when you `cat`, for example). IOW, this is to fix the red mark on the diff: 
![image](https://github.com/user-attachments/assets/f1370222-eca4-4a9a-86ad-fb20bcf527c2)

Reply:
Ok, i see, thanks.